tailwind.config.js

```diff
@@ -16,6 +16,11 @@ module.exports = {
   ],
   theme: {
     extend: {
+      colors: {
```

Comment:
What is this for? Should it belong to a different PR?

Reply:
ref: https://tailwindcss.com/docs/gradient-color-stops#customizing-your-theme, the background color of sticky header need gradient color, it's need to be defined in the `tailwind.config.js`

Reply:
> What is this for? Should it belong to a different PR?

ref: https://tailwindcss.com/docs/gradient-color-stops#customizing-your-theme, the background color of sticky header need gradient color, it's need to be defined in the tailwind.config.js